### PR TITLE
Switch to Twison Twine format and enhance the importer

### PR DESCRIPTION
**Pull request by grahamearley**
**This PR migrates the importer from the [TwineJson](https://github.com/cauli/TwineJson) Twine format to the [Twison](https://github.com/lazerwalker/twison) Twine format** (in response to #93).

It also allows users to select the destination for their story prefab (instead of just putting it in the `Assets/Ignored/` directory.  Also, the prefab is now named after the title of the Twine story, instead of just being called `Story.prefab`.

Here's what the new importer looks like:

There are a few other things to note about this change:
* **You no longer use a .json file to import the story** -- now you copy and paste the json code. This is because the Twison exporter simply creates a page with all the json text and the easiest way to import it is to copy and paste it, rather than copy and paste it into an intermediary file.
* **We now support "alias" child names for children nodes.** For example, in one of the complex workhouse Twine stories, all the nodes are titled things like `Righteous01` or `6:00am`. When those nodes are linked to, though, they use the Twine syntax `[[Get up and get ready->Righteous01]]`. This links to the twine node titled `Righteous01`, but it does so with a link that says "Get up and get ready." I added support for these child-name "aliases" in this PR.

### CODE REVIEW

File: Project/Assets/Editor/Twine Importer/TwineImporterUI.cs [hunk 1]
```diff
@@ -18,21 +19,9 @@ public class TwineImporterUI : EditorWindow
...
-			if (Path.GetExtension (filePath).Contains ("json"))
-			{
-				selectedFile = (TextAsset) Selection.activeObject;
-			}
-		}
-
 		// create and show window
 		var window = EditorWindow.GetWindow<TwineImporterUI> ();
-		window.targetFile = selectedFile;
+		window.Show ();
```
Comment (cpimhoff) [inline Project/Assets/Editor/Twine Importer/TwineImporterUI.cs:25]: This is more explicit (and thus probably clearer), but just for the record, `EditorWindow.GetWindow` automatically shows the window if it isn't in front.

File: Project/Assets/Editor/Twine Importer/TwineImporterUI.cs [hunk 2]
```diff
@@ -42,43 +31,40 @@ void OnGUI ()
 	{
 		GUILayout.Label ("Import Twine Data", EditorStyles.boldLabel);
 
+		GUILayout.Label ("Twine JSON text:");
+	
+		this.jsonString = GUILayout.TextArea (this.jsonString, GUILayout.MinHeight(20), GUILayout.MaxHeight(50));
```
Comment (cpimhoff) [inline Project/Assets/Editor/Twine Importer/TwineImporterUI.cs]: My big worry with presenting a big empty text area is that it _implies_ that users are expected to type in JSON.

I understand the new exporter doesn't add to a file, but providing a import from file options, a placeholder that says "paste in JSON from ....", or even a button that automatically replaces the contents of the text area with the clipboard, would all go a long way to making it clear that users _should under no circumstances be writing out JSON_.

**Review by cpimhoff — APPROVED** [review]
This is great for now! Happy to be escaping the broken JSON exporter.

**Review by brennan724 — APPROVED** [review]
I agree with what Charlie said about the big text area, but otherwise it looks good!

**Comment by grahamearley** [comment]
TODO:
* Put placeholder text that suggests that you copy-paste and don't write json by hand.
* Paste from clipboard button (if possible)

**Comment by grahamearley** [comment]
Okay I fixed up some things (brace yourself for huge screenshots):
* Default text in JSON text area that suggests you copy and paste into it
* A little JSON validation to make sure you're putting the right format into the textbox. Looks like this when it's invalid: 
* A paste from clipboard button! Works perfectly! 
* It turns out that we were making the first twine node in the import list be the start node. Well, that's not always how a story is created! Now we explicitly use the `startnode` field from the JSON to find the right node to activate first.
* Various cleanups

**Review by Donahuec — APPROVED** [review]
Those changes are beautiful! The checking for valid json is really nice.

**Pull request merged**